src/tools/cephfs/JournalTool.cc

```diff
@@ -85,21 +92,24 @@ int JournalTool::main(std::vector<const char*> &argv)
     return -EINVAL;
   }
 
+  dout(10) << "JournalTool::main " << dendl;
   std::vector<const char*>::iterator arg = argv.begin();
 
+  dout(10) << "JournalTool::main " << dendl;
   std::string rank_str;
   if(!ceph_argparse_witharg(argv, arg, &rank_str, "--rank", (char*)NULL)) {
     // Default: act on rank 0.  Will give the user an error if they
     // try invoking this way when they have more than one filesystem.
     rank_str = "0";
   }
 
+  dout(10) << "JournalTool::main rank_str=" << rank_str << dendl;
```

Comment:
Yes, I don't know what this is either. If you want, I can mark the code that I've written with comments or such for you to know what I've written. My code is mostly localized to the area where I have commented from where I have set an update interval, as you have mentioned below: